lib/tasks/book2.rake

```diff
@@ -79,7 +79,8 @@ def genbook (code, &get_content)
   alldoc = Nokogiri::HTML(html)
   number = 1
 
-  book = Book.where(:edition => 2, :code => code).first_or_create
+  book = Book.where(:edition => 2, :code => code).destroy_all
```

Comment:
I think this can be shortened to:

```ruby
Book.destroy_all(edition: 2, code: code)
book = Book.create(...)
```

Reply:
Yeah, skipping the `where` is nicer. I left the named parameters as `:key => value`, since that matches the rest of the code. It might be worth modernizing the whole thing if that's more idiomatic, but I think it should be done consistently.

Reply:
Oops, I'm incompetent at using `git push`. I merged this without the cleanup, which I pushed straight to master in 313dfb7d1890f5f9e52e7e6f51a268f20810f236. Yolo!